app/models/article.rb

```diff
@@ -0,0 +1,6 @@
+class Article < ApplicationRecord
+  MAX_TITLE_LENGTH = (5..40).freeze
```

Comment:
Here you uses range, but constant name say to us only about MAX. Should be used name like TITLE_LENGTH_RANGE